### PR TITLE
Fixes #10722: Use normation based vm in rtf

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/10722

### CODE REVIEW

File: vagrant.rb [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
 $ubuntu10_04 = "bento/ubuntu-10.04"
 $ubuntu12_04 = "ubuntu/precise64"
-$ubuntu12_10 = "chef/ubuntu-12.10"
+$ubuntu12_10 = "normation/ubuntu-12.04"
```
Comment (amousset) [inline vagrant.rb]: not the right one

File: vagrant.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@
 #####################################################################################
 
 # find more : https://atlas.hashicorp.com/boxes/search
-$centos5 = "hfm4/centos5"
+$centos5 = " normation/centos-5-64"
```
Comment (amousset) [inline vagrant.rb]: space at the beginning of the name

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

File: rtf [hunk 3]
```diff
@@ -253,11 +253,11 @@ if Vagrant.has_plugin?("vagrant-cachier")
   config.cache.scope = :box
 end
 
-# Mac OS X workwaround
-config.vm.provision "fix-no-tty", type: "shell" do |s|
-    s.privileged = false
-    s.inline = "sudo sed -i '/tty/!s/mesg n/tty -s \\\\&\\\\& mesg n/' /root/.profile 2>/dev/null || true"
-end
+## Mac OS X workwaround
```
Comment (VinceMacBuche) [inline rtf]: Why commenting this ? This allow to use on mac os and should only be uncommented if this is the case ?

Comment (peckpeck) [inline rtf]: It is run in every guest.
Since we have windows guest that fail with this and we never had any macosx guest let's say it is useless.

Comment (peckpeck) [inline rtf]: Moreover it is already done in cleanbox.sh

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**